src/frontend/classic/components/Viewer.svelte

```diff
@@ -96,20 +170,33 @@
                         <p>...loading image</p>
                     {:then content}-->
                     {#if mode === "Thumbnail"}
+                        <img
+                            src={placeholderImage}
+                            alt=""
+                            class="thumbnail-image-util"
+                            use:processImage={index}
```

Comment:
why not use on:load instead of use if there is no additional behavior extended  ?

Reply:
It's absolutely right